src/utils/crypto_utils/CMakeLists.txt

```diff
@@ -9,6 +9,10 @@ target_include_directories (${target_name} PUBLIC inc)
 
 find_package (OpenSSL REQUIRED)
 
+message(OPENSSL_CRYPTO_LIBRARY="${OPENSSL_CRYPTO_LIBRARY}")
```

Comment:
We need to delete these. I messed up and didn't remove them in the initial commit (it's useful for tracking across different plats but we should just do it once). 

Might be good to move to the project definition instead of here in a target definition (will only print once).

Reply:
done in new pr branch